server/admin/reload.js

```diff
@@ -24,6 +24,7 @@ var safeParse = require('../../lib/util.js').safeParse;
 
 module.exports = function createReloadHandler(ringpop) {
     return function handleReload(arg1, arg2, hostInfo, callback) {
+        ringpop.logger.warn('endpoint is deprecated: /admin/reload', {hostInfo: hostInfo});
```

Comment:
I would suggest changing the logging message to:

```
ringpop.logger.warn('ringpop endpoint is deprecated: /admin/reload', {
    local: ringpop.whoami(),
    hostInfo: hostInfo
});
```

The differences are including "ringpop" in the message itself. Since Ringpop is embedded in apps and its loggers are shared, it's typically not apparent whether the app or Ringpop is logging the message. The other difference is to include the `local` parameter as a logger metadata option because it's nice to know which Ringpop node is logging the message.